### PR TITLE
OSDOCS#12186: Remove the mention of replicas:0 in the nodepool

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.17+

Issue: https://issues.redhat.com/browse/OSDOCS-12186

Link to docs preview: https://82760--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-disconnected/hcp-deploy-dc-bm.html#hcp-nodepool-hc_hcp-deploy-dc-bm

QE/SME review:
- [x] QE/SME has approved this change.

Additional information:

### CODE REVIEW

File: modules/hcp-nodepool-hc.adoc [hunk 1]
```diff
@@ -33,15 +33,15 @@ spec:
     image: registry.<dns.base.domain.name>:5000/openshift/release-images:4.x.y-x86_64 \// <5>
   replicas: 0
```
Comment (jparrill) [inline modules/hcp-nodepool-hc.adoc:34]: You need to change this one too.

File: modules/hcp-nodepool-hc.adoc [hunk 2]
```diff
@@ -33,15 +33,15 @@ spec:
...
 status:
-  replicas: 0 // <6>
+  replicas: 2 // <6>
```
Comment (jparrill) [inline modules/hcp-nodepool-hc.adoc:36]: the `// <6>` should go into the above one in the `spec` section not in the `status` one. Status should show the same number as spec.

**Comment by xenolinux** [comment]
At this very moment - this PR can be cherry picked to 4.17 and 4.18 only. The PR updates the newly migrated topic (from ACM to OCP) which is not yet added in versions < 4.17. Hence, removing the labels for versions < 4.17 and skipping to create manual CPs for the same.

**Pull request merged**